### PR TITLE
Better readme for UI Automator

### PR DESCRIPTION
**Pull request by marcosdiez**
After working a few weeks with UI Automator I added some insights to the Readme which would make other people's life much easier. 

Now it's my chance to share it with others!

### CODE REVIEW

**Comment by marcosdiez** [comment]
I signed it!

**Comment by marcosdiez** [comment]
I signed it!

File: uiautomator/BasicSample/README.md [hunk 1]
```diff
@@ -2,7 +2,26 @@
...
+are being run on a API 18+ device.
+
+This fact has interesting consequences:
+
+- it allows the test app to be compliled quick, because the main app won't have to be compiled every time the tests are run.
+- it allows white box testing, for the source code of the app to be tested is not necessary for tests to be written nor run.
+
+
+This project uses the Gradle build system.
+To test it using the command line, simple type:
```
Comment (otw) [inline uiautomator/BasicSample/README.md]: To run tests from command line, use:

**Comment by otw** [comment]
Hey Marcos, thanks for this pull request. We already have an internal version with better phrasing which is similar to yours. This should be released with the next sample push. To keep track of the issue, can you please open up an issue for this here. I will close this pull request and update the issue. Thanks again for your time and contribution to testing samples.

**Pull request closed without merging**